guides/publishing/mapping-file/README.md

```diff
@@ -0,0 +1,22 @@
+# Mapping file
+
+Each app that have blendable views should have a `<AppName>.map.md` file which contains the list of them.
+
+The file should be provided at the root of the App Warehouse package. If the app is open source, the file should be provided in the code repository, preferably at the root.
+
+Every view must be explained in the file and preferably illustrated with a screenshot.
+
+In the future, the information in this file is used for automated blending as well as for [testing purposes](https://github.com/Starcounter/Guidelines/issues/26).
+
+Sample `Images.map.md` ([source](https://github.com/StarcounterApps/Images/blob/develop/Images.map.md)):
+
+```txt
```

Comment:
txt is not supported by prism:
```
Failed to load prism syntax: txt
{ Error: Cannot find module 'prismjs/components/prism-txt.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at requireSyntax (C:\Users\Erlend\Documents\Docs\node_modules\gitbook-plugin-prism\index.js:31:3)
    at Object.code (C:\Users\Erlend\Documents\Docs\node_modules\gitbook-plugin-prism\index.js:82:11)
    at Record.TemplateBlock.applyBlock (C:\Users\Erlend\.gitbook\versions\3.2.2\lib\models\templateBlock.js:205:23)
    at C:\Users\Erlend\.gitbook\versions\3.2.2\lib\output\getModifiers.js:56:33
    at C:\Users\Erlend\.gitbook\versions\3.2.2\lib\output\modifiers\highlightCode.js:47:24
    at C:\Users\Erlend\.gitbook\versions\3.2.2\lib\output\modifiers\editHTMLElement.js:11:16 code: 'MODULE_NOT_FOUND' }
```

Just remove "txt" and it will look the same.